src/Http/Http.Extensions/gen/StaticRouteHandlerModel/WellKnownTypeData.cs

```diff
@@ -7,19 +7,39 @@ internal static class WellKnownTypeData
 {
     public enum WellKnownType
```

Comment:
Internal?

Reply:
I made the other classes internal even though I don't think it matters much in a source generator/analzyer. This one I didn't bother with since it's inside of `internal static class WellKnownTypeData`, and I want to move this to the top level in another PR anyway.